examples/simple4.py

```diff
@@ -0,0 +1,19 @@
+import spotipy
+import os
+from pprint import pprint
+
+
+def main():
+    auth_manager = spotipy.SpotifyOAuth(
+        client_id=os.getenv("SPOTIPY_CLIENT_ID"),
+        client_secret=os.getenv("SPOTIPY_CLIENT_SECRET"),
+        redirect_uri=os.getenv("SPOTIPY_REDIRECT_URI"),
+        username=os.getenv("SPOTIPY_CLIENT_USERNAME"),
```

Comment:
If you get some free time, it would be amazing to make those optional (like for Client Credentials), so we could just do:

    spotipy.Spotify(auth_manager=spotipy.SpotifyOAuth())

Reply:
Yep, I had the idea too! I'll make this soon 👍